### PR TITLE
Add the ServiceCard component

### PR DESCRIPTION
**Pull request by BelfDev**
# ServiceCard component

**Mobile**
Screen Shot 2020-04-11 at 10 12 47 AM

**Desktop**
Screen Shot 2020-04-11 at 10 12 14 AM

- Create the `ServiceCard` component
- Add the proper styles
- Add new icons: `smartphone` | `growth` | `monitor` | `sketch`
- Add a new shadow color named `color-card-shadow-soft` (#0000001F)
- Fix style vs styles naming convention → using `styles` from now on

### CODE REVIEW

File: src/resources/icons/smartphone-ic.svg [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  <defs>
+    <style>.a{fill:url(#a);}</style>
+    <linearGradient id="a" x1="1.523" y1="-0.307" x2="-0.704" y2="1.27" gradientUnits="objectBoundingBox">
+      <stop offset="0" stop-color="#0f6"/>
+      <stop offset="1" stop-color="#0090ff"/>
+    </linearGradient>
+  </defs>
+  <g transform="translate(-97.35)">
+    <g transform="translate(97.35)">
+      <path class="a"
```
Comment (yagotome) [inline src/resources/icons/smartphone-ic.svg:12]: @BelfDev this kind of class I told you that may bring us conflicts between different svgs. We need to re-export svgs in a way that every styles are inline.

File: src/resources/icons/smartphone-ic.svg [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="23.623" height="39.226"
+     viewBox="0 0 23.623 39.226">
+  <defs>
+    <style>.a{fill:url(#a);}</style>
+    <linearGradient id="a" x1="1.523" y1="-0.307" x2="-0.704" y2="1.27" gradientUnits="objectBoundingBox">
```
Comment (yagotome) [inline src/resources/icons/smartphone-ic.svg:5]: I've been experiencing conflict issues due to `linearGradient`s using same `id` such as this does.

File: src/resources/icons/index.js [hunk 3]
```diff
@@ -4,6 +4,7 @@ const Icons = {
   letter: require('./letter-ic.svg'),
   downArrow: require('./down-arrow-ic.svg'),
   gatsby: require('./gatsby-icon.png'),
+  smartphone: require('./smartphone-ic.svg'),
```
Comment (yagotome) [inline src/resources/icons/index.js:7]: Where are all the other introduced icons?

File: src/resources/icons/smartphone-ic.svg (hunk 1 above)
Comment (BelfDev) [reply src/resources/icons/smartphone-ic.svg:12]: I'll have to check with Vini. This SVG has been exported straight out of Adobe XD.

File: src/resources/icons/smartphone-ic.svg (hunk 2 above)
Comment (BelfDev) [reply src/resources/icons/smartphone-ic.svg:5]: Wow

File: src/resources/icons/index.js (hunk 3 above)
Comment (BelfDev) [reply src/resources/icons/index.js:7]: @yagotome I was going to add them later...

**Pull request merged**